server/bin/utils/run-unittest

```diff
@@ -580,8 +581,29 @@ function _run_test {
     _dump_logs
     _verify_output ${testname}
     _reset_state ${testname}
-    return 0
+
+    pass_or_fail=$(cat ${_testres})
+    if [[ ${pass_or_fail} == "PASS" ]]; then
+        echo "${pass_or_fail} - ${testname} ($(cat ${_testdur}))"
+        rm ${_testres}
+        rm -f ${_testdur}
+        rm -f ${_test_db}
+        rmdir ${_testroot}
```

Comment:
Do you really want all of these in separate commands?  Given that this is the `PASS` case, wouldn't `rm -rf ${_testroot}` do what we want?  If not, can't we combine the first three into a single command?...why does the first omit the `-f`?

Reply:
We wanted to see when things were left around or something was missing.  In particular, the `rmdir` should be cleaning up an empty directory.